packages/OCanren-ppx/OCanren-ppx.0.3.0/opam

```diff
@@ -0,0 +1,68 @@
+opam-version: "2.0"
+synopsis:
+  "Implementation of miniKanren relational (logic) EDSL: PPX extensions"
+description: """
+PPX rewriters for writing relational programs more conveniently.
+
+1) `ppx_fresh` -- a clone of original miniKanren syntax
+
+  fresh (x...) goal
+
+expands into
+
+  Fresh.numeral (fun x ... -> goal)
+
+2) `ppx_distrib` for easier generation of type-safe smart constructors and reifiers...
+
+  [%%distrib
+    type nonrec _ t = fully_abstract_type
+    type ground = ...
+  ]
+
+3) `ppx_repr` for testing. Expands `REPR(expr)` to `(string_of_expr expr, expr)`
+"""
+maintainer: ["Dmitrii Kosarev Kakadu@pm.me"]
+authors: [
+  "Dmitrii Kosarev https://github.com/Kakadu"
+  "JetBrains Research https://research.jetbrains.org"
+]
+license: "LGPL-2.1"
+homepage: "https://github.com/JetBrains-Research/OCanren"
+bug-reports: "https://github.com/JetBrains-Research/OCanren/issues"
+depends: [
+  "dune" {>= "2.9" }
+  "ocaml" {>= "4.10"}
+  "dune-configurator"
+  "ppxlib" {>= "0.22" & <= "0.24"}
```

Comment:
This prevents installation with ocaml 4.14. Is that expected?

Reply:
It's installable on 4.14 from now